### ISSUE TITLE
OS X installation fails with Numpy CFLAGS missing

### ISSUE THREAD

**Issue by joanwa** [mdtraj/mdtraj#1143]
```
mdtraj/formats/xtc/xtc.c:283:10: fatal error: 'numpy/arrayobject.h' file not found
    #include "numpy/arrayobject.h"
             ^
    1 error generated.
```

One needs to manually export the CFLAGS e.g.

export -CFLAGS=‘-I /usr/local/lib/python3.5/site-packages/numpy/core/include $CFLAGS'

**Comment by marscher** [comment]
what is the output of:
``` bash
python -c "import numpy; print(numpy.get_include())"
```

This directory is being used by mdtraj setup. If it can not be found it is very likely that your numpy installation/Python path is broken somehow.

**Comment by mpharrigan** [comment]
This might be fixed by #1290

**Comment by mpharrigan** [comment]
please try the new release of mdtraj 1.9 and re-open if the issue persists
